packages/react-chat/src/views/ChatWidget/index.tsx

```diff
@@ -19,7 +20,7 @@ interface ChatWidgetProps extends React.PropsWithChildren {
 
 const ChatWidget: React.FC<ChatWidgetProps> = ({ chatAPI, ready }) => {
   const { assistant, open, close, interact } = useContext(RuntimeStateAPIContext);
-  const { isOpen } = useContext(RuntimeStateContext);
+  const { isOpen, shadowRoot } = useContext(RuntimeStateContext);
```

Comment:
I think if there is anything we should check inject as a prop, it should be the shadowRoot instead of putting it in the context. I don't think it makes sense to have a DOM element in the context, and would like to keep it more pure (same reason why you wouldn't have any DOM in redux state).

Seems very easy as it is only used in the top level
https://github.com/voiceflow/react-chat/blob/553fb3664029e8ad03ff561031ee050d2105ecc5/packages/react-chat/src/index.tsx#L67-L68

We can probably remove this line after too:
https://github.com/voiceflow/react-chat/blob/a3233a6e188504f10c5cf24cb325f8856e965968/packages/react-chat/src/contexts/RuntimeContext/index.tsx#L18

Reply:
moved it to a prop 👍